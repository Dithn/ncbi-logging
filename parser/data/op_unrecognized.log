13.59.252.14 - - [07/Jun/2020:01:09:44 -0400] "sra-download.ncbi.nlm.nih.gov" "GET ..\x5C..\x5C..\x5C..\x5C..\x5C..\x5C..\x5C..\x5C..\x5C..\x5Cwindows\x5Cwin.ini HTTP/1.1" 400 150 0.012 "-" "-" "-" port=80 rl=0
13.59.252.14 - - [07/Jun/2020:01:09:45 -0400] "sra-download.ncbi.nlm.nih.gov" "GET /cgi-bin/d6rw1r5y.asp?<script>document.cookie=%22testtifh=7678;%22</script> HTTP/1.1" 301 162 0.000 "-" "Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0; NIHInfoSec)" "-" port=80 rl=390
61.153.216.106 - - [07/Jun/2020:00:04:05 -0400] "sra-download.ncbi.nlm.nih.gov" "GET /traces/sra47/SRR/010462/SRR10713958 HTTP/1.1" 206 34758384 1959.489 "https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10713958#" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36 Edg/83.0.478.45" "-" port=443 rl=459
165.112.6.3 - - [07/Jun/2020:00:06:24 -0400] "sra-download.ncbi.nlm.nih.gov" "GET /" 400 248 0.000 "-" "-" "-" port=443 rl=7
13.78.48.192 - - [07/Jun/2020:00:22:21 -0400] "sra-download.ncbi.nlm.nih.gov" "GET /traces/sra9/SRR/009095/SRR9313661 HTTP/2.0" 206 78718882 266.729 "-" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3314.0 Safari/537.36 SE 2.X MetaSr 1.0" "-" port=443 rl=54
165.112.6.3 - - [07/Jun/2020:00:22:24 -0400] "sra-download.ncbi.nlm.nih.gov" "GET /" 400 248 0.000 "-" "-" "-" port=443 rl=7
61.153.216.106 - - [07/Jun/2020:00:22:26 -0400] "sra-download.ncbi.nlm.nih.gov" "GET /traces/sra47/SRR/010462/SRR10713958 HTTP/1.1" 206 454383 201.680 "https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10713958#" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36 Edg/83.0.478.45" "-" port=443 rl=460
158.111.236.250 - - [01/Jan/2020:02:50:24 -0500] "sra-download.ncbi.nlm.nih.gov" "GET /traces/sra34/SRR/003923/SRR4017927 HTTP/1.1" 206 32768 0.000 "-" "linux64 sra-toolkit fastq-dump.2.9.1" "-" port=443 rl=293
139.80.16.229 - - [01/Jan/2020:02:50:24 -0500] "sra-download.ncbi.nlm.nih.gov" "GET /traces/sra32/SRR/005807/SRR5946882 HTTP/1.1" 206 32768 0.000 "-" "linux64 sra-toolkit fastq-dump.2.9.3" "-" port=443 rl=293
139.80.16.229 - - [01/Jan/2020:02:50:24 -0500] "sra-download.ncbi.nlm.nih.gov" "GET /traces/sra32/SRR/005807/SRR5946882 HTTP/1.1" 206 32768 0.000 "-" "linux64 sra-toolkit fastq-dump.2.9.3" "-" port=443 rl=293
139.80.16.229 - - [01/Jan/2020:02:50:24 -0500] "GET /traces/sra32/SRR/005807/SRR5946882 HTTP/1.1" 206 32768 0.000 "-" "linux64 sra-toolkit fastq-dump.2.9.3" "-" port=443 rl=293
nonesense nonesense
139.80.16.229 - - [01/Jan/2020:02:50:24 -0500] "GET /traces/sra32/SRR/005807/SRR000006 HTTP/1.1" 206 32768 0.000 "-" "linux64 sra-toolkit fastq-dump.2.9.3" "-" port=443 rl=293
blah 139.80.16.229 - - [01/Jan/2020:02:50:24 -0500] "GET /traces/sra32/SRR/005807/SRR000007 HTTP/1.1" 206 32768 0.000 "-" "linux64 sra-toolkit fastq-dump.2.9.3" "-" port=443 rl=293
139.80.16.229 - - [01/Jan/2020:02:50:24 -0500] "GET /traces/sra32/SRR/005807/SRR00008?a=b HTTP/1.1" 206 32768 0.000 "-" "linux64 sra-toolkit fastq-dump.2.9.3" "-" port=443 rl=293
139.80.16.229
