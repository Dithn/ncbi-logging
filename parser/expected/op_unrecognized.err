converting on-premise format
total : 34
accepted : 30
rejected : 2
unrecognized : 2
tool-version : 1.0.0
compiled at : Jul 23 2020 - 11:33:29
