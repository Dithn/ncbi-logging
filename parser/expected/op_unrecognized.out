{"accepted":true,"agent":"linux64 sra-toolkit test-sra.2.8.2","forwarded":"-","ip":"159.226.149.175","line_nr":1,"port":443,"referer":"-","req_len":164,"req_time":"0.000","request":{"method":"HEAD","path":"/srapub/SRR5385591.sra > SRR5385591.sra.out 2>&1","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":404,"res_len":0,"source":"NCBI","time":"15.8.2018:10:31:47 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"10.154.195.11","line_nr":2,"port":80,"referer":"-","req_len":0,"req_time":"0.001","request":{"method":"GET","path":"","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":400,"res_len":150,"source":"NCBI","time":"7.6.2020:8:32:5 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"13.56.21.202","line_nr":3,"port":80,"referer":"-","req_len":0,"req_time":"0.071","request":{"method":"","path":"\\x16\\x03\\x01\\x02\\x00\\x01\\x00\\x01\\xFC\\x03\\x03\\xD5\\xAC\\xA1\\xD1r\\x85\\x91)\\x92v\\xB0p\\x038B#\\xD9\\x007\\x08\\xCB\\x86\\x08\\x8C\\x1Cs\\x04 \\x99\\x8A\\xC0\\x12 a\\xD7\\xC9C\\xEE\\x93\\x91\\xBF\\xCEASvP\\x9F\\xDB\\x91\\x85\\x9D\\xB4\\xF7Ee\\x80%^\\xBD\\xF3\\x02\\xB4\\x00w\\xBE\\x00\\xD2\\x13\\x02\\x13\\x03\\x13\\x01\\xC0,\\xC00\\x00\\xA3\\x00\\x9F\\xCC\\xA9\\xCC\\xA8\\xCC\\xAA\\xC0\\xAF\\xC0\\xAD\\xC0\\xA3\\xC0\\x9F\\xC0]\\xC0a\\xC0W\\xC0S\\x00\\xA7\\xC0+\\xC0/\\x00\\xA2\\x00\\x9E\\xC0\\xAE\\xC0\\xAC\\xC0\\xA2\\xC0\\x9E\\xC0\\x5C\\xC0`\\xC0V\\xC0R\\x00\\xA6\\xC0$\\xC0(\\x00k\\x00j\\xC0s\\xC0w\\x00\\xC4\\x00\\xC3\\x00m\\x00\\xC5\\xC0#\\xC0'\\x00g\\x00@\\xC0r\\xC0v\\x00\\xBE\\x00\\xBD\\x00l\\x00\\xBF\\xC0","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":400,"res_len":150,"source":"NCBI","time":"7.6.2020:22:26:43 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"13.56.21.202","line_nr":4,"port":80,"referer":"-","req_len":0,"req_time":"1.071","request":{"method":"","path":"","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":400,"res_len":0,"source":"NCBI","time":"7.6.2020:22:27:55 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"34.201.223.169","line_nr":5,"port":80,"referer":"-","req_len":0,"req_time":"0.008","request":{"method":"","path":"\\x16\\x03\\x01\\x02\\xAB\\x01\\x00\\x02\\xA7\\x03\\x03^\\xDC\\xAA\\x06Gdy8XrYvMhcCxPHEdMshpIQBaBRe\\x00\\x024\\x00\\xA2\\x00\\xA3\\xC0\\x80\\xC0\\x81\\xC0\\xA6\\x00\\xAA\\xC0\\xA7\\x00\\xAB\\xC0\\x96\\xC0\\x90\\xC0\\x97\\xC0\\x91\\xCC\\xAD\\xC0\\x9E\\xC0\\xA2\\x00\\x9E\\xC0\\x9F\\xC0\\xA3\\x00\\x9F\\xC0|\\xC0}\\xCC\\xAA\\x00\\xA4\\x00\\xA5\\xC0\\x82\\xC0\\x83\\x00\\xA0\\x00\\xA1\\xC0~\\xC0\\x7F\\x00\\xA6\\x00\\xA7\\xC0\\x84\\xC0\\x85\\xC0\\xAC\\xC0\\xAE\\xC0+\\xC0\\xAD\\xC0\\xAF\\xC0,\\xC0r\\xC0\\x86\\xC0s\\xC0\\x87\\xCC\\xA9\\xC0\\x9A\\xC0\\x9B\\xCC\\xAC\\xC0/\\xC00\\xC0v\\xC0\\x8A\\xC0w\\xC0\\x8B\\xCC\\xA8\\xC0-\\xC0.\\xC0t\\xC0\\x88\\xC0u\\xC0\\x89\\xC01\\xC02\\xC0x\\xC0\\x8C\\xC0y\\xC0\\x8D\\xC0\\xAA\\xC0\\xAB\\xC0\\xA4\\xC0\\xA8\\x00\\xA8\\xC0\\xA5\\xC0\\xA9\\x00\\xA9\\xC0\\x94\\xC0\\x8E\\xC0\\x95\\xC0\\x8F\\xCC\\xAB\\x00\\xAC\\x00\\xAD\\xC0\\x98\\xC0\\x92\\xC0\\x99\\xC0\\x93\\xCC\\xAE\\xC0\\x9C\\xC0\\xA0\\x00\\x9C\\xC0\\x9D\\xC0\\xA1\\x00\\x9D\\xC0z\\xC0{\\x00c\\x00e\\x00\\x11\\x00\\x13\\x002\\x008\\x00D\\x00\\x87\\x00\\x12\\x00f\\x00\\x99\\x00\\x8F\\x00\\x90\\x00\\x91\\x00\\x8E\\x00\\x14\\x00\\x16\\x003\\x009\\x00E\\x00\\x88\\x00\\x15\\x00\\x9A\\x00\\x0B\\x00","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":400,"res_len":150,"source":"NCBI","time":"7.6.2020:4:49:11 -400","user":""}
{"accepted":true,"agent":"Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0; NIHInfoSec)","forwarded":"-","ip":"34.201.223.169","line_nr":6,"port":80,"referer":"-","req_len":319,"req_time":"0.000","request":{"method":"","path":"HCHALE / HTTP/1.1","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":405,"res_len":552,"source":"NCBI","time":"7.6.2020:4:49:29 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"10.154.195.11","line_nr":7,"port":80,"referer":"-","req_len":8,"req_time":"0.001","request":{"method":"GET","path":"/","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":301,"res_len":162,"source":"NCBI","time":"7.6.2020:8:32:5 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"10.154.195.11","line_nr":8,"port":80,"referer":"-","req_len":0,"req_time":"0.000","request":{"method":"GET","path":"/","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/3.14"},"res_code":505,"res_len":180,"source":"NCBI","time":"7.6.2020:8:32:5 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"10.154.195.11","line_nr":9,"port":80,"referer":"-","req_len":0,"req_time":"0.001","request":{"method":"GET","path":"/","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/"},"res_code":400,"res_len":150,"source":"NCBI","time":"7.6.2020:8:32:5 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"10.154.195.11","line_nr":10,"port":80,"referer":"-","req_len":0,"req_time":"0.000","request":{"method":"","path":"get / http/1.0","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":400,"res_len":150,"source":"NCBI","time":"7.6.2020:8:32:5 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"13.59.252.14","line_nr":11,"port":80,"referer":"-","req_len":0,"req_time":"0.011","request":{"method":"","path":"\\x16\\x03\\x01\\x00\\xCF\\x01\\x00\\x00\\xCB\\x03\\x01","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":400,"res_len":150,"source":"NCBI","time":"7.6.2020:1:9:52 -400","user":""}
{"accepted":true,"agent":"Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0; NIHInfoSec)","forwarded":"-","ip":"13.59.252.14","line_nr":12,"port":80,"referer":"-","req_len":376,"req_time":"0.000","request":{"method":"PROPFIND","path":"/","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":405,"res_len":552,"source":"NCBI","time":"7.6.2020:1:10:23 -400","user":""}
{"accepted":true,"agent":"Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0; NIHInfoSec)","forwarded":"-","ip":"13.59.252.14","line_nr":13,"port":80,"referer":"-","req_len":336,"req_time":"0.000","request":{"method":"GET","path":"/login/?user=|\\x22`id`\\x22|","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":301,"res_len":162,"source":"NCBI","time":"7.6.2020:1:14:11 -400","user":""}
{"accepted":true,"agent":"","forwarded":"-","ip":"146.118.64.48","line_nr":14,"port":443,"referer":"-","req_len":117,"req_time":"0.000","request":{"method":"HEAD","path":"/traces/sra19/SRR/009763/SRR9997476","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":200,"res_len":0,"source":"NCBI","time":"7.6.2020:2:47:10 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"34.201.223.169","line_nr":15,"port":80,"referer":"-","req_len":0,"req_time":"0.013","request":{"method":"","path":"\\x16\\x03\\x01\\x00\\xCF\\x01\\x00\\x00\\xCB\\x03\\x01\\x1D\\x018\\xD1L}!\\xFC\\x1FiEv\\x12\\xE8\\x1Fk\\x9F\\xE7\\x0E<\\x22~\\xAA$G=\\x1B\\x8E%\\x1E\\x15\\xA9\\x00\\x00R\\xC0","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":400,"res_len":150,"source":"NCBI","time":"7.6.2020:4:49:9 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"13.59.252.14","line_nr":16,"port":80,"referer":"-","req_len":0,"req_time":"0.012","request":{"method":"GET","path":"..\\x5C..\\x5C..\\x5C..\\x5C..\\x5C..\\x5C..\\x5C..\\x5C..\\x5C..\\x5Cwindows\\x5Cwin.ini","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":400,"res_len":150,"source":"NCBI","time":"7.6.2020:1:9:44 -400","user":""}
{"accepted":true,"agent":"Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0; NIHInfoSec)","forwarded":"-","ip":"13.59.252.14","line_nr":17,"port":80,"referer":"-","req_len":390,"req_time":"0.000","request":{"method":"GET","path":"/cgi-bin/d6rw1r5y.asp?<script>document.cookie=%22testtifh=7678;%22</script>","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":301,"res_len":162,"source":"NCBI","time":"7.6.2020:1:9:45 -400","user":""}
{"accepted":true,"agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36 Edg/83.0.478.45","forwarded":"-","ip":"61.153.216.106","line_nr":18,"port":443,"referer":"https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10713958#","req_len":459,"req_time":"1959.489","request":{"method":"GET","path":"/traces/sra47/SRR/010462/SRR10713958","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":206,"res_len":34758384,"source":"NCBI","time":"7.6.2020:0:4:5 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"165.112.6.3","line_nr":19,"port":443,"referer":"-","req_len":7,"req_time":"0.000","request":{"method":"GET","path":"/","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":400,"res_len":248,"source":"NCBI","time":"7.6.2020:0:6:24 -400","user":""}
{"accepted":true,"agent":"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3314.0 Safari/537.36 SE 2.X MetaSr 1.0","forwarded":"-","ip":"13.78.48.192","line_nr":20,"port":443,"referer":"-","req_len":54,"req_time":"266.729","request":{"method":"GET","path":"/traces/sra9/SRR/009095/SRR9313661","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/2.0"},"res_code":206,"res_len":78718882,"source":"NCBI","time":"7.6.2020:0:22:21 -400","user":""}
{"accepted":true,"agent":"-","forwarded":"-","ip":"165.112.6.3","line_nr":21,"port":443,"referer":"-","req_len":7,"req_time":"0.000","request":{"method":"GET","path":"/","server":"sra-download.ncbi.nlm.nih.gov","vers":""},"res_code":400,"res_len":248,"source":"NCBI","time":"7.6.2020:0:22:24 -400","user":""}
{"accepted":true,"agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36 Edg/83.0.478.45","forwarded":"-","ip":"61.153.216.106","line_nr":22,"port":443,"referer":"https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10713958#","req_len":460,"req_time":"201.680","request":{"method":"GET","path":"/traces/sra47/SRR/010462/SRR10713958","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":206,"res_len":454383,"source":"NCBI","time":"7.6.2020:0:22:26 -400","user":""}
{"accepted":true,"agent":"linux64 sra-toolkit fastq-dump.2.9.1","forwarded":"-","ip":"158.111.236.250","line_nr":23,"port":443,"referer":"-","req_len":293,"req_time":"0.000","request":{"method":"GET","path":"/traces/sra34/SRR/003923/SRR4017927","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":206,"res_len":32768,"source":"NCBI","time":"1.1.2020:2:50:24 -500","user":""}
{"accepted":true,"agent":"linux64 sra-toolkit fastq-dump.2.9.3","forwarded":"-","ip":"139.80.16.229","line_nr":24,"port":443,"referer":"-","req_len":293,"req_time":"0.000","request":{"method":"GET","path":"/traces/sra32/SRR/005807/SRR5946882","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":206,"res_len":32768,"source":"NCBI","time":"1.1.2020:2:50:24 -500","user":""}
{"accepted":true,"agent":"linux64 sra-toolkit fastq-dump.2.9.3","forwarded":"-","ip":"139.80.16.229","line_nr":25,"port":443,"referer":"-","req_len":293,"req_time":"0.000","request":{"method":"GET","path":"/traces/sra32/SRR/005807/SRR5946882","server":"sra-download.ncbi.nlm.nih.gov","vers":"HTTP/1.1"},"res_code":206,"res_len":32768,"source":"NCBI","time":"1.1.2020:2:50:24 -500","user":""}
{"accepted":true,"agent":"linux64 sra-toolkit fastq-dump.2.9.3","forwarded":"-","ip":"139.80.16.229","line_nr":26,"port":443,"referer":"-","req_len":293,"req_time":"0.000","request":{"method":"GET","path":"/traces/sra32/SRR/005807/SRR5946882","server":"","vers":"HTTP/1.1"},"res_code":206,"res_len":32768,"source":"NCBI","time":"1.1.2020:2:50:24 -500","user":""}
{"unrecognized":"nonesense nonesense"}
{"accepted":true,"agent":"linux64 sra-toolkit fastq-dump.2.9.3","forwarded":"-","ip":"139.80.16.229","line_nr":28,"port":443,"referer":"-","req_len":293,"req_time":"0.000","request":{"method":"GET","path":"/traces/sra32/SRR/005807/SRR000006","server":"","vers":"HTTP/1.1"},"res_code":206,"res_len":32768,"source":"NCBI","time":"1.1.2020:2:50:24 -500","user":""}
{"unrecognized":"unrecognized 139.80.16.229 - - [01/Jan/2020:02:50:24 -0500] \"GET /traces/sra32/SRR/005807/SRR000007 HTTP/1.1\" 206 32768 0.000 \"-\" \"linux64 sra-toolkit fastq-dump.2.9.3\" \"-\" port=443 rl=293"}
{"accepted":true,"agent":"linux64 sra-toolkit fastq-dump.2.9.3","forwarded":"-","ip":"139.80.16.229","line_nr":30,"port":443,"referer":"-","req_len":293,"req_time":"0.000","request":{"method":"GET","path":"/traces/sra32/SRR/005807/SRR00008?a=b","server":"","vers":"HTTP/1.1"},"res_code":206,"res_len":32768,"source":"NCBI","time":"1.1.2020:2:50:24 -500","user":""}
{"accepted":false,"agent":"","forwarded":"","ip":"139.80.16.229","line_nr":31,"port":0,"referer":"","req_len":0,"req_time":"","request":{"method":"","path":"","server":"","vers":""},"res_code":0,"res_len":0,"source":"NCBI","time":"0.0.0:0:0:0 0","unparsed":"139.80.16.229","user":""}
{"accepted":false,"agent":"","forwarded":"","ip":"139.80.16.229","line_nr":32,"port":0,"referer":"","req_len":0,"req_time":"","request":{"method":"","path":"","server":"","vers":""},"res_code":0,"res_len":0,"source":"NCBI","time":"0.0.0:0:0:0 0","unparsed":"139.80.16.229 - - [01/Jan/2020:02:50:24 -0500] ","user":""}
